LeaderboardBackend.Test/Modships.cs

```diff
@@ -35,41 +36,111 @@ public static async Task MakeMod_Success()
 		Leaderboard createdLeaderboard = await CreateLeaderboard(jwt);
 
 		// Make user a mod
-		Modship created = await ApiClient.Post<Modship>(
-			"/api/modships",
+		Modship created = await CreateModship(createdLeaderboard.Id, jwt);
+
+		// Confirm that user is a mod
+		Modship retrieved = await GetModship(jwt);
+
+		Assert.NotNull(created.User);
+		Assert.AreEqual(created, retrieved);
+	}
+
+	[Test]
+	public static async Task RemoveMod_Success()
+	{
+		string jwt = (await ApiClient.LoginAdminUser()).Token;
+
+		Leaderboard createdLeaderboard = await CreateLeaderboard(jwt);
+
+		// Make user a mod
+		Modship created = await CreateModship(createdLeaderboard.Id, jwt);
+
+		// Confirm that user is a mod
+		Modship retrieved = await GetModship(jwt);
+
+		// Remove the modship
+		await RemoveModship(createdLeaderboard.Id, jwt);
+
+		try
```

Comment:
I think instead of returning from the catch block and throwing an exception after, it would be better to `Assert.Fail` as the second line of the try block. With `Assert.Fail` you can actually give it a nicer message to describe the expected behaviour too.